lib/tasks/server/livereload-server.js

```diff
@@ -40,7 +40,7 @@ module.exports = Task.extend({
   start: function(options) {
     var tlroptions = {};
     tlroptions.ssl = options.ssl || false;
-    tlroptions.host = '0.0.0.0';
+    tlroptions.host = options.liveReloadHost || '0.0.0.0';
```

Comment:
isn't the code in the run method enough? As i can't see how i can default to host here, since host itself isn't decided yet.